src/main/java/com/bobocode/parser/PropertyParser.java

```diff
@@ -0,0 +1,16 @@
+package com.bobocode.parser;
+
+import java.util.Map;
+
+/**
+ * Parses a .yml file into a {@link Map Map<String, String>}, where key of type {@link String} represents <b>property name</b>,
+ * and value of type {@link String} represents <b>property value</b>
+ * A file may contain multiple blocks separated by the <b>---</b> sign (3 dashes). Provide a block name with the <b>persistence-unit.name</b> property
```

Comment:
Hopefully we don't force those --- lines and it's only an option

Reply:
I've changed the Javadoc here